stories/assembly-gcp-upgrade.adoc

```diff
@@ -0,0 +1,50 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-gcp-upgrade"]
+
+:context: gcp-upgrade
+
+= Upgrading your deployment
+
+You can upgrade your existing {AAPonGCP} deployment to the newer version. 
+The upgrade process covers upgrading of {HubName} and {ControllerName}. 
+The upgrade process takes roughly the same amount of time as a {AAPonGCP} deployment install. 
+
+Use the following procedure to ensure a smooth upgrade process.
+
+[NOTE]
+=====
+{AAPonGCP} supports sequential upgrades. 
+All upgrades should be no more than one major version behind the version you are currently upgrading to. 
+For example, to upgrade to {PlatformNameShort} from {GCPMarket} to 2.3.20230221, you must be on version 2.2.20230215.
```

Comment:
NOTE: The version change below should also go in the AWS upgrade doc!

From
For example, to upgrade to {PlatformNameShort} from {GCPMarket} to 2.3.20230221, you must be on version 2.2.20230215.
To 
For example, to upgrade to {PlatformNameShort} from {GCPMarket} to 2.3.20230221-00, you must be on version 2.2.20230215-00.